api/watson_service.py

```diff
@@ -0,0 +1,27 @@
+import json
+import os
+import requests
+from dotenv import load_dotenv, find_dotenv
+from django.http import JsonResponse
+from ibm_watson import ToneAnalyzerV3
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
+load_dotenv(find_dotenv())
+
+def analyze_tone(mood_text):
+  authenticator = IAMAuthenticator(os.getenv("WATSON_APIKEY"))
```

Comment:
So no need to create another .env file, it works if you just add the apikey to the current .env file

Reply:
https://help.pythonanywhere.com/pages/environment-variables-for-web-apps/